linera-base/src/identifiers.rs

```diff
@@ -47,23 +48,23 @@ pub struct Account {
     pub chain_id: ChainId,
     /// The owner of the account, or `None` for the chain balance.
     #[debug(skip_if = Option::is_none)]
-    pub owner: Option<Owner>,
+    pub owner: Option<AccountOwner>,
```

Comment:
I believe so, but this needs a closer look. I think #2496 would have to be done first though. Or it could actually be done as a part of it :thinking: